website/docs/icons/library/partials/code.md

```diff
@@ -0,0 +1,9 @@
+### Hds::Icon component
```

Comment:
Should this file be named `code.md` or `resources.md` since its just linking to different kinds of information?

Reply:
We tent do use the same name as the "tab", if possible. In this case the tab says `Code` because, as developer, when I see it I know that I have to click there. If I see `Resources` I may not realize there's something below that is of interest for me.

Reply:
That's fair. Ok, ill resolve since you know best what folks will be looking for.